examples/index.json

```diff
@@ -1014,7 +1014,7 @@
         "column": 3,
         "category": "Working over the network",
         "samples": [{
-                "name": "TCP Socket Basic Example",
+                "name": "Basic TCP Socket",
```

Comment:
We seem to have several other "Xxx Basic Example" titled BBEs. Shall we rename them too?

Reply:
Fixed the remaining socket example as well. Stdlib team will revisit the remaining samples and will provide meaningful names. @wggihan Please note.